routers/user/home.go

```diff
@@ -520,45 +559,74 @@ func Issues(ctx *context.Context) {
 		}
 	}
 
-	issueStats, err := models.GetUserIssueStats(models.UserIssueStatsOptions{
+	userIssueStats, err := models.GetUserIssueStats(models.UserIssueStatsOptions{
 		UserID:      ctxUser.ID,
 		UserRepoIDs: userRepoIDs,
 		FilterMode:  filterMode,
 		IsPull:      isPullList,
 		IsClosed:    isShowClosed,
 	})
 	if err != nil {
-		ctx.ServerError("GetUserIssueStats", err)
+		ctx.ServerError("GetUserIssueStats User", err)
```

Comment:
?

Reply:
I thought it added more context since there are multiple calls to GetUserIssueStats. But I can remove it if you want, I don't feel that strongly about it.

Reply:
me neither ... but if a error mesage gets more uniqe its better I think